source/Services/atlassian.html.md.erb

```diff
@@ -0,0 +1,198 @@
+---
+title: Atlassian Apps - Jira, Confluence, Crowd and GlusterFS
+last_reviewed_on: 2025-01-17
+review_in: 12 months
+weight: 159
+---
+
+# <%= current_page.data.title %>
+
+We have deployed Atlassian apps Jira, Confluence, Crowd on the staging environment. The [repository](https://github.com/hmcts/atlassian-infrastructure) has all the IaC as well as automation scripts.
+
+Our goal was to create replica of the production atlassian environment on the staging environment.
+
+This documentation provides a detailed outline of the procedure, offering guidance for anyone tasked with rolling out atlassian apps from scratch.
+
+## Prerequisites
+
+The atlassian production environment is deployed on the [MOJ DCD Atlassian LVE](https://portal.azure.com/#@HMCTS.NET/resource/subscriptions/79898897-729c-41a0-a5ca-53c764839d95/overview) subscription and the staging environment is deployed on the [MOJ DCD Atlassian NLE](https://portal.azure.com/#@HMCTS.NET/resource/subscriptions/b7d2bd5f-b744-4acc-9c73-e068cec2e8d8/overview) subscription. so make sure you have access to these subscriptions as contruibutor.
+
+
+## Existing Production Setup
+
+Please take a look at this [documentation](https://tools.hmcts.net/confluence/pages/viewpage.action?spaceKey=DTSPO&title=ODP+0002+-+Atlassian+Interim+Hosting+Deployment+Approach) on confluence which has detailed list of VMs and existing setup and the diagram.
+
+There are total 9 VMs in the production environment, 3 VMs for Jira, 2 VMs for Confluence, 1 VM for Crowd and 3 VMs for GlusterFS.
+
+All the VMs are on this resource group [RG-PRD-ATL-INT-01](https://portal.azure.com/#@HMCTS.NET/resource/subscriptions/79898897-729c-41a0-a5ca-53c764839d95/resourceGroups/RG-PRD-ATL-INT-01/overview).
+
+#### Jira VMs
+* PRDATL01AJRA01.cp.cjs.hmcts.net
+* PRDATL01AJRA02.cp.cjs.hmcts.net
+* PRDATL01AJRA03.cp.cjs.hmcts.net
+
+#### Confluence VMs
+* PRDATL01ACNF02.cp.cjs.hmcts.net
+* PRDATL01ACNF04.cp.cjs.hmcts.net
+
+#### Crowd VM
+* PRDATL01ACRD01.cp.cjs.hmcts.net
+
+#### GlusterFS VMs
+* PRDATL01DGST01.cp.cjs.hmcts.net
+* PRDATL01DGST02.cp.cjs.hmcts.net
+* PRDATL01DGST03.cp.cjs.hmcts.net
+
+The database is available on the [RG-PRD-ATL-01](https://portal.azure.com/#@HMCTS.NET/resource/subscriptions/79898897-729c-41a0-a5ca-53c764839d95/resourceGroups/RG-PRD-ATL-01) resource group.
+
+#### Postgres Single Server
+ps-prd-atl-dpdb
+
+Please note that there are other important resources like Application Gateways, Load Balancer, NSGs, VNETs, etc. which are not mentioned here.
+
+## Approach for Staging deployment
+
+We decided to use the same setup as production for the staging environment. We have first of all used backup copies of the above VMs to deploy the VMs on staging. We have also used the same database server for the staging environment.
```

Comment:
This might be unclear, makes it sound like the staging servers are using the prod database. Might be better to say we restored a backup of the production database to a new staging single server.